test/ini_files/blue.ini

```diff
@@ -9,7 +9,10 @@ system_bucket = elasticbeanstalk-fourfront-webprod-system
 accession_factory = encoded.server_defaults.enc_accession
 elasticsearch.server = search-fourfront-blue-xkkzdrxkrunz35shbemkgrmhku.us-east-1.es.amazonaws.com:80
 snovault.app_version = ask-pip
+env.bucket = foursight-envs
+env.ecosystem = main
 env.name = fourfront-blue
+env.name.bs = fourfront-blue
```

Comment:
In theory all these files can be removed except the any.ini versions (which should be aligned to the versions in cgap-portal)

Reply:
Yeah, I'll remove those later when I'm sure things are in a working state and I have time to consider a simpler replacement